messaging/tck/README.adoc

```diff
@@ -25,22 +25,26 @@ This project contains the TCK for MicroProfile Reactive Messaging.
 
 The TCK uses http://arquillian.org/[Arquillian], so an implementation needs Arquillian container support to run. In addition, the implementation must also implement the TCK's SPI in order to allow the TCK to interact and make assertions on messages sent and received from topics under test.
 
-=== `TckContainer`
+=== Optional features
 
-The primary entry point into the tck is the `org.eclipse.microprofile.reactive.messaging.tck.spi.TckContainer`. An instance of this must be provided through the JDK `ServiceLoader` mechanism (that is, provide a `META-INF/services/org.eclipse.microprofile.reactive.messaging.tck.spi.TckContainer` file that contains the fully qualified classname of the implementation).
+It is not mandatory to implement all features of the spec. The TCK defines TestNG groups for each feature. If a particular implementation (or a particular messaging provider being tested by the TCK) doesn't support a particular feature of the spec, it can exclude that feature from being tested by creating a TestNG test suite descriptor that includes the `all` group but excludes the group for that particular feature. Alternatively, an implementation can selectively enable the groups for the features that it does support. Currently, the groups defined are:
```

Comment:
I disagree this. All features of the spec should be implemented. Otherwise, how can I guarantee app's portability?